models.py

```diff
@@ -152,6 +152,13 @@ def get_full_recipe_from_id(cls, id):
         recipe['ingredients'] = ingredients
         return recipe
 
+    @classmethod
+    def return_recipe_lengths(cls, ids):
+        '''
+        Takes recipe id, returns a tuple with the id and name
+        '''
+        recipe_lengths = [(id, len(cls.query.filter(Recipe.id==id).first().ingredients)) for id in ids]
```

Comment:
Readability alert - I don't understand what's happening here.

Reply:
The function docstring was wrong. I updated it to better reflect what's going on.